app/js/pages/DappRequestPage.js

```diff
@@ -0,0 +1,183 @@
+'use strict'
+
+import React, { Component } from 'react'
+import DocumentTitle from 'react-document-title'
+import StickyBox from 'react-sticky-box'
```

Comment:
I'm curious, do we need to add a new dependency for something that just applies:
```css
position: -webkit-sticky;
position: sticky;
top: 0;
``` 
to a div?

Reply:
I've used this historically since it offers support for IE and Edge, which `position: sticky` does not. At least it used to -- I see the author of `react-sticky-box` added this note to the latest release:
```
Browsers with no sticky support will fallback to a position: "relative" behaviour which in my opinion is much preferrable to the half-broken situation before for older browsers. You still might want to check out the 0.5 branch in case you need to support stickyness in IE and Edge <= 15.
```

So good catch, thanks!!

Reply:
@aulneau I updated this and also removed the horizontal rules from below each section -- they were a bit awkward since the width of each section was narrow.